src/app/pages/admin-page/tables/offer-table/offer-table.component.scss

```diff
@@ -0,0 +1,72 @@
+@import '/src/app/styles/variables.scss';
+
+.table {
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  display: flex;
+  flex-direction: column;
+  box-shadow: none;
+
+  &__body {
+    overflow: auto;
+    flex: 1 0 1px;
+
+    .full-width-table {
+      width: 100%;
+      min-width: 600px;
+
+      &::ng-deep .mat-cell {
+        padding-right: 0.25em;
+        &:first-of-type {
+          padding-left: 1em;
+        }
+      }
+    }
+  }
+
+  &__header,
+  &__footer {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 0 1em;
+    background-color: #fff;
+  }
+
```

Comment:
&__header {
  @media screen and (max-width: $mobile) {
      flex-wrap: wrap;
      justify-content: center;
  }
}